src/Resources/config/routing.yml

```
@@ -1,33 +1,30 @@
 simple_things_entity_audit_home:
     path: /{page}
+    controller: SimpleThings\EntityAudit\Controller\AuditController::index
```

```suggestion
    controller: SimpleThings\EntityAudit\Controller\AuditController::indexAction
```
Shouldnt I add full method name? @sonata-project/contributors